test/cljs/rems/administration/test_create_form.cljs

```diff
@@ -171,6 +171,8 @@
                              :field/index 0
                              :field/title {:en "en title"
                                            :fi "fi title"}
+                             :field/info-text {:en "en info text"
+                                               :fi "fi info text"}
```

Comment:
I don't understand why all these have info-text defined. Surely it's not compulsory everywhere?

Reply:
My comment was about that there are now many places in the tests where you have added the same lines of `info-text`. Maybe some tests shouldn't have it and some should. After all it's not compulsory.

Reply:
now there's a test case both ways

Reply:
Resolved